var/spack/repos/builtin/packages/nf-core-tools/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class NfCoreTools(PythonPackage):
+    """A python package with helper tools for the nf-core community."""
+
+    homepage = "https://nf-co.re/tools"
+    pypi = "nf-core/nf-core-2.7.1.tar.gz"
+    maintainers = ["marcodelapierre"]
+
+    version("2.7.1", sha256="90de62390314ef3141cee700667f017aa65c0346e40704a0f70d0662abcfb0db")
+    version("2.6", sha256="47d4df906a60006249284bbf0bb84cdec48303a699c7c0d0a26f404a50e0811a")
+    version("2.5.1", sha256="0303f6e3810ba1bc6ac843566ee9dea6b5edbf3527437dec5854b8c437456a4f")
+
+    depends_on("nextflow@21.10.3:", type="run")
+
+    depends_on("python@3.7:3", type=("build", "run"))
+    depends_on("py-setuptools@40.6:", type="build")
+
+    depends_on("py-click", type=("build", "run"))
+    depends_on("py-filetype", type=("build", "run"))
+    # depends_on("py-galaxy-tool-util", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-galaxy-tool-util", type=("build", "run"))
```

Reply:
I had commented out because some dependency recipes are still missing for that specific dependency.

I am actually going to remove the `nf-core-tools` recipe from this PR, I will add it back at the end after adding all its dependencies.